mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/AnnotationHandlerMapping.java

```diff
@@ -21,9 +27,45 @@ public AnnotationHandlerMapping(final Object... basePackage) {
 
     public void initialize() {
         log.info("Initialized AnnotationHandlerMapping!");
+        Reflections reflections = new Reflections(basePackage);
+
+        Set<Class<?>> typesAnnotatedWith = reflections.getTypesAnnotatedWith(Controller.class);
+
+        for (Class<?> aClass : typesAnnotatedWith) {
+
+            Object handler;
+            try {
+                handler = aClass.getDeclaredConstructor().newInstance();
+            }catch (Exception e) {
```

Comment:
```suggestion
            } catch (Exception e) {
```

사소 하지만, 정렬 설정 해두는 걸 추천해요!